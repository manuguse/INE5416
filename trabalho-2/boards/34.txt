2 - 3 7 - 5 3 - - 1
- - 5 - 3 - - 4 - 7
- - - - - - - - - -
- - - - - 5 - - 5 -
6 - - 1 - - - 3 - 3
4 - 5 2 - 4 - - - -
- 2 - 1 - - 5 - 5 -
7 6 - 3 - 6 - - - 5
2 - 2 - 3 4 1 - - 3
1 - - - - 6 - - - -