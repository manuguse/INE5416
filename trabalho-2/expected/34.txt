2 5 3 7 4 5 3 6 4 1
1 4 5 6 3 7 1 4 2 7
2 3 4 3 4 6 2 3 1 5
1 2 1 2 1 5 4 2 5 4
6 5 7 1 2 1 2 3 1 3
4 3 5 2 7 4 1 4 6 2
1 2 4 1 6 3 5 3 5 1
7 6 1 3 2 6 3 2 1 5
2 5 2 4 3 4 1 7 4 3
1 2 1 2 1 6 7 1 2 1